Files/Converters/InverseBoolConverter.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using Windows.UI.Xaml.Data;
+
+namespace Files.Converters
```

Comment:
Is it needed?

Reply:
The namespace or the Converter itself?

The Converter was being used before I was asked to remove the ShowDotfileNames setting. It's not being used anymore but I left it alone incase it's useful in future.

Reply:
You can remove it as there is default BoolNegationConverter exists.

Reply:
Removed.